aws/requests.yaml

```diff
@@ -1,5 +1,5 @@
 releases:
-    - name: "> 15.2.1"
+    - name: "> 15.2.2"
```

Comment:
Is this right? Aren't those dependencies still valid for 15.2.1? (or have I misunderstood)

Reply:
The reason why I bumped it was simply because we had to update all the components, currently cert-manager is broken. In order to release it quickly without including all the components I simply bumped the version.

Reply:
inside the requests.yaml we add components which needed to be higher than a certain version if you would create a new release.